management/build.gradle

```diff
@@ -20,3 +20,9 @@ dependencies {
   compile project(':api'), project(':core'), project(':impl')
   compile group: 'org.terracotta.management', name: 'management-registry', version: parent.managementVersion
 }
+
+repositories {
+  mavenCentral()
+  maven { url "http://repo.terracotta.org/maven2" }
+  maven { url "http://snapshots.terracotta.org/" }
```

Comment:
Yes it make sense... We'll need to open an issue thought to make sure we are ready for central publication...